### PR TITLE
Build containers based on RedHat UBI

### PR DESCRIPTION
**Pull request by ilfrin**

### CODE REVIEW

File: presto/Dockerfile.ubi [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM starburstdata/ubi8-python2
```
Comment (kokosing) [inline presto/Dockerfile.ubi]: missing lincense

File: presto/Dockerfile.ubi [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+LABEL maintainer="https://www.starburstdata.com/"
+
+ARG presto_version=302-e.7
+ARG dist_location=""
+
+COPY ./installdir /installdir
+
+RUN yum -y install java-1.8.0-openjdk less && \
```
Comment (kokosing) [inline presto/Dockerfile.ubi]: put each program in separate line

File: ubi/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Please refer to your agreement(s) with Starburst for further information.
+################################################################################
+
+FROM registry.redhat.io/ubi8/ubi
+LABEL maintainer="https://www.starburstdata.com/"
+
+# explicitly installing python3 prior to python2 so that the a later python3 install would not delete the python symlink created below
+RUN set -eux && \
+    yum install -y yum-utils && \
+    yumdownloader python2 && \
```
Comment (kokosing) [inline ubi/Dockerfile]: avoid using `yumdownloader `, prefer curl

File: ubi/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,36 @@
+################################################################################
```
Comment (sopel39) [inline ubi/Dockerfile]: I would maybe put it in https://github.com/starburstdata/docker-images-private/tree/master/manual

**Comment by sopel39** [comment]
Why this wasn't merged @ilfrin . I think it needs to be updated now with respect to non-ubi changes.

**Pull request closed without merging**